x-pack/plugins/threat_intelligence/public/components/paywall/paywall.stories.tsx

```diff
@@ -14,5 +14,5 @@ export default {
 };
 
 export function BasicPaywall() {
```

Comment:
nit: replace occurrences of `BasicPaywall` with `Paywall`

Reply:
I'd rather not change another team's component name in this PR :)